examples/browser/package.json

```diff
@@ -40,7 +41,7 @@
     "clean": "theia clean && rimraf errorShots",
     "build": "theia build --mode development",
     "watch": "yarn build --watch --mode development",
-    "start": "theia start",
+    "start": "theia start --root-dir=../workspace",
```

Comment:
`--root-dir` is deprecated, use `theia start ../workspace`

please don't hardcode options in the npm script, they should be provided to it, e.g from the cli `yarn start ../workspace`